### ISSUE TITLE
Syslog logging not possible on Xenial with unprivileged user

### ISSUE THREAD

**Issue by beyhan** [cloudfoundry/bosh-linux-stemcell-builder#82]
We observe that we cannot write to syslog on Xenial stemcells with a non root 
user belonging to no group.

*Trusty > 3586.x*
```
zookeeper/df901ee6-87b2-4a83-8aec-8b68574f0f98:~# su - testuser
zookeeper/df901ee6-87b2-4a83-8aec-8b68574f0f98:~$ whoami
testuser
zookeeper/df901ee6-87b2-4a83-8aec-8b68574f0f98:~$ logger message
zookeeper/df901ee6-87b2-4a83-8aec-8b68574f0f98:~$
```

*Xenial > 170.x*
```
bosh/cec4c995-5f80-4b41-8262-f604b9db1a97:~$ su - testuser
bosh/cec4c995-5f80-4b41-8262-f604b9db1a97:~$ whoami
testuser
bosh/cec4c995-5f80-4b41-8262-f604b9db1a97:~$ logger testmessage
logger: socket /dev/log: Permission denied
```

***Steps to reproduce***
1. log into a vm using Xenial as stemcell
2. create a new user, e.g. `adduser testuser`
3. switch to that user, `su - testuser`
4. execute `logger <message>`
5. fails with `logger: socket /dev/log: Permission denied`

Please note that this is not reproducible on `bosh-lite`

**Comment by ansd** [comment]
On trusty:
```
$ ls -l /dev/log
srw-rw-rw- 1 root root 0 Feb 25 19:23 /dev/log
```

On xenial:
```
$ ls -l /dev/log
lrwxrwxrwx 1 root root 28 Mar  8 16:22 /dev/log -> /run/systemd/journal/dev-log

$ ls -l /run/systemd/journal/dev-log
srw-rw---- 1 root vcap 0 Mar  8 15:50 /run/systemd/journal/dev-log
```
Therefore, only the root user and the vcap group can write to syslog on xenial stemcells.

The default for xenial (non bosh stemcells) is that everyone can write to syslog:
```
$ ls -l /run/systemd/journal/dev-log
srw-rw-rw- 1 root root 0 Mar  8 11:09 /run/systemd/journal/dev-log
```
So the question is where in the linux-stemcell-builder are these file permissions modified and why?

**Comment by s4heid** [comment]
@beyhan I guess this can be closed now, since #84 has been merged.

**Comment by beyhan** [comment]
Yes, we missed to close the issue automatically with the pr.
